contexts/PanelContext.tsx

```diff
@@ -10,7 +10,7 @@ export interface PanelContextState {
 }
 
 export const defaultPanelContextState = {
-  panelType: null,
+  panelType: "menu" as const,
```

Comment:
having a default panel type instead of null makes life easier (don't have to manage null values)